tox.ini

```diff
@@ -40,7 +40,7 @@ deps =
     dj18: Django>=1.8,<1.9
     dj110: Django>=1.10,<1.11
     dj111: Django>=1.11a1,<2.0
-    master: https://github.com/django/django/tarball/master
+    dj20: Django<2.1
```

Comment:
Please keep master in the tox config.

Reply:
@blueyed is tox.ini used by circleci?

Reply:
Yes, but not all of it - only selected envs, via the TOXENV variable.

Reply:
And it is useful to still being able to run it locally, `tox -e py36-djmaster`.

Reply:
Ahh, _now_ why you request "master" to stay. It doesn't affect the CI builds, but is handy for local testing. Thanks @blueyed.